### PR TITLE
Create compact view

### LINKED ISSUE #188: Compact View
So that users don't have to click 2 buttons to step into a day.

### PR DESCRIPTION
**Pull request by Graywing13**
Closes #188

### CODE REVIEW

**Review by NLmeng — APPROVED** [review]

File: client/tailwind.config.js [hunk 1]
```diff
@@ -26,6 +31,14 @@ module.exports = {
           position: 'absolute',
           'z-index': 10,
         },
+        '.city-skyline': {
+          background:
+            "url('https://static.vecteezy.com/system/resources/previews/008/045/334/original/silhouette-of-the-city-city-skyline-silhouette-modern-cityscape-for-t-shirt-abstract-city-landscape-illustration-free-vector.jpg')",
```
Comment (L0Lmaker) [inline client/tailwind.config.js:36]: Not the biggest fan of the background image, thought my screen was glitching at first 😅. Really like the colors on hover though 🤩

**Comment by L0Lmaker** [comment]
Couple of questions:
* Will compact view be an option for mobile? If so what will it look like?
* Should we remove the See more panel in compact view?
* How hard would it be to reorient the map to fit the pins nicely on the left side? (aesthetics only)

File: client/tailwind.config.js (hunk 1 above)
Comment (Graywing13) [reply client/tailwind.config.js:36]: hmm I see. any suggestions for picture themes? just the og background seemed a bit plain

**Comment by Graywing13** [comment]
1. Didn't really consider it on mobile... since the teacups and the show more panel are in the same category, and afaik we are removing the teacups. 
2. They serve slightly different purposes: the see more panel doesn't show descriptions
3. what do you mean by pins? can you screenshot please?

**Comment by L0Lmaker** [comment]
All right, looks good to me then 👍

**Pull request merged**